src/cmd/serial.js

```diff
@@ -966,23 +984,23 @@ class SerialCommand {
 			});
 		});
 
-		const parseSecurityType = (ent, cb) => {
+		const parsesecurity = (ent, cb) => {
 			resetTimeout();
-			if (securityType) {
+			if (opts.security) {
 				let security = 3;
-				if (securityType.indexOf('WPA2') >= 0 && securityType.indexOf('802.1x') >= 0) {
```

Comment:
Your code looks good and works, this is just a random idea.

Not sure 100% if it works, but it looks kinda cool

```
if (opts.security) {
  const s = opts.security.reduce((acc, current) => { return Object.assign({}, acc, { [current]: true })}, {});

  security = security || (s['WPA2'] && s['802.1x'] && 5);
  isEnterprise = security || isEnterprise;

  security = security || (s['WPA'] && s['802.1x'] && 4);
  isEnterprise = security || isEnterprise;

  security = security || (s['WEP'] && 3);
  isEnterprise = security || isEnterprise;
}
```

Reply:
This is not my code. It doesn't have any tests so I'd rather leave it alone.